VERSION

```diff
@@ -4,5 +4,5 @@
 #  OTEL_VERSION=<otel version> make update-otel-version
 #  VERSION=<manifests version> make update-manifests-version
 
-MANIFESTS_VERSION=0.2.0
+MANIFESTS_VERSION=0.2.1
```

Comment:
Why the patch version change as opposed to a minor version upgrade?

Reply:
The major change here is a config change to use OTLP exporters instead of GoogleCloud exporters for Trace.

Wasn't sure about the semantics here. Do you think because GBOC version was updated we should move it to minor version update ?

Reply:
On second thought, a minor version bump might better reflect the nature of these changes.